src/envoy/auth/jwt.cc

```diff
@@ -31,6 +31,7 @@
 namespace Envoy {
 namespace Http {
 namespace Auth {
+namespace {
 
 namespace Base64url {
```

Comment:
So we need either rename `decode()` (of base64url) to `base64UrlDecode()`, or modify all its call to `Auth::decode()` to avoid collision. The former is better?